wiki/People/The_Team/pl.md

```diff
@@ -1,61 +1,71 @@
-| [Sinistro](https://osu.ppy.sh/u/5530) | Meeting minutes, Ogromny wkład w FAQ |
-| [woc2006](https://osu.ppy.sh/u/1105845) | Jeden z twórców gry, Twórca osu!manii |
+| [![][flag_US] awp](https://osu.ppy.sh/users/2650) ([Barrel Roll Weekly](http://brw.twinkfish.com/)) | Generalna pomoc, projekt szkicu chibi |
+| [![][flag_US] Derekku](https://osu.ppy.sh/users/91341) | Generalna pomoc, zarządzanie społecznością |
+| [![][flag_NZ] Echo](https://osu.ppy.sh/users/431) ([blog](http://blog.echo.sh/)) | Programista, zintegrowanie IRC z czatem gry, prowadzenie strony osu! |
+| [![][flag_NL] Intermezzo](https://osu.ppy.sh/users/136842) | Programista, twórca osz2/p2p |
+| ![][flag_US] Jim ([Brand New Games](http://www.bravegamer.com/)) | Oryginalny projekt strony |
+| [![][flag_DE] Loctav](https://osu.ppy.sh/users/71366) | Organizacja oficjalnych turniejów, członek administracji, przewodzenie grupie QAT oraz zarządzanie społecznością |
+| [![][flag_US] LuigiHann](https://osu.ppy.sh/users/1079) ([LuigiHann@Deviantart](https://luigihann.deviantart.com/)) | Projekt skórki, wkład w domyślną skórkę oraz zestaw ikon |
+| [![][flag_US] mm201](https://osu.ppy.sh/users/30655) | Programista, twórca sliderów mm |
+| [![][flag_US] nuudles](https://osu.ppy.sh/users/21312) | Rozwijanie osu! na iPhone |
+| ![][flag_US] Phil | Projekt forum |
+| [![][flag_US] Sarumaru](https://osu.ppy.sh/users/9427)  ([Sarumaru@deviantART.com](https://sarumaru.deviantart.com/)) | Oryginalny projekt i szkic Pippi |
+| [![][flag_GR] Sinistro](https://osu.ppy.sh/users/5530) | Zarządzanie społecznością, organizowanie spotkań, wkład w FAQ |
+| [![][flag_CN] woc2006](https://osu.ppy.sh/users/1105845) | Programista, rozwijanie osu!manii |
+| [![][flag_US] ztrot](https://osu.ppy.sh/users/6347) | Profesor [akademii osu!](/wiki/Announcements/osu!academy), projekt niektórych maskotek |
 
 Grupy
-----------------------
+-------
 
-| Pseudonim | Opis |
+| Nazwa | Opis |
 | ---- | ----------- |
-| [Moderatorzy globalni](/wiki/People/Global_Moderation_Team) | Sprawowanie porządku na forum i w czacie |
-| [Nominatorzy Beatmap](/wiki/People/Beatmap_Nomination_Group) | Osoby które zrobią wszystko, żeby twoja mapa była "zakwalifikowana" |
-| [Twórcy](http://osu.ppy.sh/g/11) | Tworzenie wspaniałej gry poprzez naprawianie bugów i dodawanie nowych opcji |
-| [osu! alumni](/wiki/People/osu!_Alumni) | Ludzie zapisani w historii osu! |
-| [Support Team Redux](/wiki/People/Support_Team) | Pomoc z prośbami nt. nowych rzeczy |
-| [osu!news](http://osu.ppy.sh/g/25) | Zapewnianie dobrej lektury od 2013 |
-| [Staff Turniejowy](http://osu.ppy.sh/g/26) | Ludzie odpowiadający za prowadzenie oficjalnych mistrzostw świata w osu! |
-| [Zespół zapewnienia jakości](/wiki/People/Quality_Assurance_Team) | Zapewniamy jakość beatmap, abyś mógł się dobrze bawić. |
+| [Moderatorzy globalni](/wiki/People/Global_Moderation_Team) | Sprawowanie nadzoru na forum i czatem |
```

Comment:
na -> nad

Reply:
w sumie "nad czatem i forum" brzmi lepiej przy czytaniu na głos, ale to już jest jeszcze bardziej kosmetyczne niż rozwijanie -> rozwój